metadata-proxy/nginx.conf

```diff
@@ -0,0 +1,47 @@
+user www-data;
+worker_processes 4;
+pid /run/nginx.pid;
+
+events {
+  worker_connections 20;
+}
+
+http {
+  server {
+    listen 127.0.0.1:2020;
```

Comment:
I don't think nginx configs support variables. In fact, I think their developers staunchly oppose that feature. IIRC, they recommend sed.

I could move it to a privileged port, and then only privileged pods w/host networking could conflict, instead of just host networking. I'll also drop this to v0.1, in case we decide the value should be configurable.